File: php/examples/buildUrls.php
<?php

// Serie can be the ID, the Name or if the series is already loaded the array
$serie = 3; // 24
$season = 1;
$episode = 1;

require_once '../src/BurningSeries.php';
$bs = new BurningSeries();

$serieUrl = $bs->buildSerieUrl($serie);
$seasonUrl = $bs->buildSeasonUrl($serie, $season);
$episodeUrl = $bs->buildEpisodeUrl($serie, $season, $episode);

// Or short:
// $serieUrl = $bs->buildUrl($serie);
// $seasonUrl = $bs->buildUrl($serie, $season);
// $episodeUrl = $bs->buildUrl($serie, $season, $episode);

echo "<a href=\"{$serieUrl}\" target=\"_blank\">Serie for {$serie} ({$serieUrl})</a> <br />";
echo "<a href=\"{$seasonUrl}\" target=\"_blank\">Season {$season} ({$seasonUrl})</a> <br />";
echo "<a href=\"{$episodeUrl}\" target=\"_blank\">Episode {$episode} ({$episodeUrl})</a>";

File: java/examples/de/jonesboard/burningseries/examples/buildUrls.java
package de.jonesboard.burningseries.examples;

import de.jonesboard.burningseries.BurningSeries;

public class buildUrls {

	public static void main(String[] args) {
		// Serie can be the ID, the Name or if the series is already loaded the array
		int serie = 3; // 24
		int season = 1;
		int episode = 1;

		BurningSeries bs = new BurningSeries();

		try {
			String serieUrl = bs.buildSerieUrl(serie);
			String seasonUrl = bs.buildSeasonUrl(serie, season);
			String episodeUrl = bs.buildEpisodeUrl(serie, season, episode);
	
			System.out.println("Serie for " + serie + ";\tLink: " + serieUrl);
			System.out.println("Season " + season + ";\tLink: " + seasonUrl);
			System.out.println("Episode " + episode + ";\tLink: " + episodeUrl);
		} catch(Exception e) {
			e.printStackTrace();
		}
	}


}
